RenderStateNotation/CMakeLists.txt

```diff
@@ -24,6 +24,12 @@ file(MAKE_DIRECTORY "${RSN_PARSER_GENERATED_HEADERS_DIR}")
 # is not accidentally used.
 file(COPY ../.clang-format DESTINATION "${RSN_PARSER_GENERATED_HEADERS_DIR}")
 
+if (${CMAKE_VERSION} VERSION_EQUAL "3.17.0" OR
+    ${CMAKE_VERSION} VERSION_GREATER "3.17.0")
+    message(STATUS "Trying to use Python3 from virtual env")
+    set(Python_FIND_VIRTUALENV FIRST)
+endif()
```

Comment:
Few comments:
- there is [`VERSION_GREATER_EQUAL`](https://cmake.org/cmake/help/latest/command/if.html#version-greater-equal) comparison operator
- I don't think there is a need to print the message as in practice it will always be printed out
- Why is version 3.17 and not 3.15 (`Python_FIND_VIRTUALENV` was added in 3.15)?

Reply:
As a matter of fact, this change should not really be needed as `FIRST` is the default behavior. Wondering why it makes difference at all:

<img width="1035" height="419" alt="image" src="https://github.com/user-attachments/assets/b4eccff7-a49c-4df1-a174-78d6229ec95a" />

https://cmake.org/cmake/help/latest/module/FindPython.html